core/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -672,17 +672,7 @@ public CommitStats commitStats() {
     }
 
     public IndexingStats indexingStats(String... types) {
-        Engine engine = getEngineOrNull();
-        final boolean throttled;
-        final long throttleTimeInMillis;
-        if (engine == null) {
-            throttled = false;
-            throttleTimeInMillis = 0;
-        } else {
-            throttled = engine.isThrottled();
-            throttleTimeInMillis = engine.getIndexThrottleTimeInMillis();
```

Comment:
Hang on: this is index throttling, which still exists and is different from store throttling?

Index throttling is when ES forces incoming indexing to be single threaded because merges have fallen too far behind ...